content/src/main/content/jcr_root/apps/acs-commons/authoring/shared-component-properties/touchui/js/shared-component-properties.js

```diff
@@ -117,4 +117,13 @@
         }
     });
 
+    // we listen to the editables to reset the Shared Component Properties Registered flag so that the Shared and
+    // Global properties are loaded as page refreshes due to Edit Config Listeners
+    channel.on('cq-editables-updated', function (ev) {
+        // reset the flag so that the shared and global icons are loaded again after page refresh
+        if (ev.editables){
```

Comment:
```suggestion
        if (ns.acsSharedComponentPropertiesIsRegistered
            && ns.EditorFrame
            && ns.EditorFrame.editableToolbar
            && ns.EditorFrame.editableToolbar._customActions) {
```

Since this bug affects only AEM 6.3 (6.4 changes how custom actions are added to the toolbar in a way that custom actions are not lost on page refresh for `afteredit`) updating this to check if this is the old 6.3 code and only set the flag then so that 6.4+ are not affected at all.

Reply:
not sure why you were testing `ev.editables` before but I took that check out.  We can add it back if necessary though.

Reply:
Above suggestion sounds good and tested in AEM 6.3.